### PR TITLE
Feature/issues/38

### PR DESCRIPTION
**Pull request by seanl-adg**
https://github.com/AdguardTeam/ExtendedCss/issues/35
https://github.com/AdguardTeam/ExtendedCss/issues/36
https://github.com/AdguardTeam/ExtendedCss/issues/37
https://github.com/AdguardTeam/ExtendedCss/issues/38

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  - grunt build
+deploy:
+  - provider: pages
+    skip_cleanup: true
+    github_token: $GITHUB_TOKEN
+    on:
+      branch: master
+  - provider: releases
+    api_key: $GITHUB_OAUTH_TOKEN
+    file: dist/extended-css.min.js
```
Comment (ameshkov) [inline .travis.yml]: I suggest getting rid of that `dist` folder in the repo and sticking to GH releases only. What I mean is that we'd better publish releases manually with beautiful release notes and such.

File: test/index.html [hunk 2]
```diff
@@ -0,0 +1,6 @@
+<ol>
```
Comment (ameshkov) [inline test/index.html]: Add a link to that page to the README so that qa guys could easily find it.

Comment (seanl-adg) [inline test/index.html]: It is already there.

File: README.md [hunk 3]
```diff
@@ -28,10 +28,19 @@ Selector property | Description
 
 ### Debugging extended selectors
 
-[Here](https://AdguardTeam.github.io/ExtendedCss/dist/bookmarklet.html) is a bookmarklet that loads ExtendedCss to the page. You can use the `ExtendedCss` constructor in a global scope, and its method `ExtendedCss.query` as `document.querySelectorAll`.
-
+To load ExtendedCss to a current page, run the following code in a browser console:
+```
+!function(E,x,t,C,s,s_){C=E.createElement(x),s=E.getElementsByTagName(x)[0],C.src=t,
+C.onload=function(){alert('ExtCss loaded successfully')},s.parentNode.insertBefore(C,s)}
```
Comment (ameshkov) [inline README.md]: `console.log` is enough

**Comment by seanl-adg** [comment]
@atropnikov @sc0rp10 could you please check whether everything is all right?

**Pull request merged**